spec/features/assignments/edit_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'edit an assignment' do
+  let(:roster) { create :roster }
+  let(:user) { create :user, rosters: [roster] }
+  let(:assignment) { create :assignment, user: user, roster: roster }
+  before :each do
+    Timecop.freeze Date.new(2018, 1, 10)
+    set_current_user(user)
+  end
+  after :each do
+    Timecop.return
+  end
+
+  it 'returns the user to the appropriate index page' do
+    date_today = Date.new(2017, 4, 4)
+    start_date = Date.new(2017, 3, 31)
+    assignment.start_date = start_date
```

Comment:
Why not set the date that way in the `let` block?

Reply:
I set some variables using `let` because I know they'll be used for future tests, I expect to use these specific dates only for this test. I can certainly could put them in a `context` block or under the `describe` block if that would be better?

Reply:
Whoops, don't mind that comment. I was looking at the wrong line.